terraform/modules/mongodb/mongodb-manifests/mongodb.yaml

```diff
@@ -0,0 +1,78 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: mongo
+  labels:
+    name: mongo
+spec:
+  ports:
+  - port: 27017
+    targetPort: 27017
+  clusterIP: None
+  selector:
+    role: mongo
+---
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: mongo
+spec:
+  serviceName: "mongo"
+  replicas: ${REPLICAS_NUMBER}
+  template:
+    metadata:
+      labels:
+        role: mongo
+        environment: test
```

Comment:
This indentation seems off. Also I think we are missing `app: mongodb` or `app: mongo` label you are using later in `podAntiAffinity`

Reply:
done